go/vt/vttablet/endtoend/misc_test.go

```diff
@@ -467,14 +467,41 @@ func TestQueryStats(t *testing.T) {
 		RowsReturned: 0,
 		ErrorCount:   1,
 	}
-	if stat != want {
-		t.Errorf("stat: %+v, want %+v", stat, want)
-	}
+	utils.MustMatch(t, want, stat)
 	vend := framework.DebugVars()
 	compareIntDiff(t, vend, "QueryCounts/vitess_a.Select", vstart, 2)
 	compareIntDiff(t, vend, "QueryRowCounts/vitess_a.Select", vstart, 0)
+	compareIntDiff(t, vend, "QueryRowsReturned/vitess_a.Select", vstart, 2)
```

Comment:
For completeness we should assert that the `QueryRowsAffected` stat does not change for a select.

Reply:
Well, that stat will not be produced at all for `Select` query type